price/price.go

```diff
@@ -17,6 +17,13 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const ETHGWeiCommaShift = 9
+const GWeiWeiCommaShift = 9
+const ETHWeiCommaShift = ETHGWeiCommaShift + GWeiWeiCommaShift
```

Comment:
**issue:** I am pretty sure that we already have the conversion values 9 and 18 for this somewhere in the code so they should not be set again here.

Please check whether it makes sense to use `ClCurrencyDecimals`, `ClCurrencyDivisor`, `ElCurrencyDecimals` and/or `ElCurrencyDivisor`.

Reply:
removed.
I missed these constants in types.Config (still learning...).